lib/netsuite.rb

```diff
@@ -9,7 +9,13 @@
 module NetSuite
   autoload :Configuration,    'netsuite/configuration'
   autoload :Response,         'netsuite/response'
-  autoload :AsyncStatus,      'netsuite/async_status'
+  autoload :Status,           'netsuite/status'
+
+  module Async
+    autoload :AsyncStatus,        'netsuite/async/async_status'
```

Comment:
Yep.  :)
On Feb 27, 2015 7:45 PM, "Jerome Cornet" <notifications@github.com> wrote:

> In lib/netsuite.rb
> <https://github.com/prburke/netsuite/pull/3#discussion_r25550607>:
>
> > @@ -9,7 +9,13 @@
> >  module NetSuite
> >    autoload :Configuration,    'netsuite/configuration'
> >    autoload :Response,         'netsuite/response'
> > -  autoload :AsyncStatus,      'netsuite/async_status'
> > +  autoload :Status,           'netsuite/status'
> > +
> > +  module Async
> > +    autoload :AsyncStatus,        'netsuite/async/async_status'
>
> should this be Async::Status instead of Async::AsyncStatus ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prburke/netsuite/pull/3/files#r25550607>.
>